firestore-bigquery-export/extension.yaml

```diff
@@ -250,6 +274,55 @@ params:
     default: NONE
     required: true
     
+
+
+  - param: TIME_PARTITIONING_FIELD
+    label: BigQuery SQL Time Partitioning table schema field(column) name
+    description: >-
+      BigQuery table column/schema field name for TimePartitioning. You can choose schema available like `timestamp` OR give a name for new BigQuery Table schema column/field that will be assigned to the selected Firestore Document field below. example: 'post_date'. If not set, the table is partitioned by pseudo column '_PARTITIONTIME'; if set, the table is partitioned by this field. Cannot be changed if Table is already partitioned.
+    type: string
+    required: false
+
+
+  - param: TIME_PARTITIONING_FIRESTORE_FIELD
+    label: Firestore Document field name for BigQuery SQL Time Partitioning field option
+    description: >-
+      This parameter will allow you to partition the BigQuery table
+      created by the extension based on selected Document Firestore field. This will generate one partition based on selected field. The Firestore Document field value must be a top-level TIMESTAMP, DATETIME, DATE field BigQuery string format or Firestore timestamp(will be converted to BigQuery TIMESTAMP). Cannot be changed if Table is already partitioned.
+      example: `postDate`
+    type: string
+    required: false
+
+
+  - param: TIME_PARTITIONING_FIELD_TYPE
+    label: BigQuery SQL Time Partitioning table schema field(column) type
+    description: >-
+      Parameter for BigQuery SQL schema field type(BigQuery SQL format) for selected Time Partitioning Firestore Document field option. If you use Firestore Timestamp in your Firestore Document it will be converted into BigQuery SQL TIMESTAMP. Cannot be changed if Table is already partitioned.
+    type: select
+    options:
+      - label: TIMESTAMP
+        value: TIMESTAMP
+      - label: DATETIME
+        value: DATETIME
+      - label: DATE
+        value: DATE
+      - label: omit
+        value: omit
+    default: TIMESTAMP
+    required: false
+
+  - param: CLUSTERING
+    label: BigQuery SQL table clustering (experimental)
+    description: >-
+      This parameter will allow you to set up Clustering for the BigQuery Table
+      created by the extension. (for example: `data,document_id,timestamp`- no whitespaces). You can select up to 4 coma separated fields(order matters). 
```

Comment:
Clarify order matters similar to the documentation change above.

Reply:
The user using Clustering should already know why it matters in my opinion. Do we want to paste and copy something from official docs?
https://cloud.google.com/bigquery/docs/clustered-tables#when_to_use_clustering